tests/e2e/e2e.test.ts

```diff
@@ -362,9 +369,11 @@ it('When bundled custom scalars are used the project type checks and generates e
           JsonManually
           DateTimeManually
           BytesManually
+          BigIntManually
           someEnumA
           someDateTimeField
           someBytesField
+          someBigIntField
```

Comment:
I would like to add an assertion down below that shows that `typeof data.bars[0].foo.someBigIntField === 'bigint'`.

I would like to do this actually for all the custom scalars. But this PR can just do the BigInt one, as you prefer.

Reply:
I will do it later

Reply:
Added, typeof someBigIntField received in the client would be string though as far as I understand

Reply:
There should be a way to setup a GraphQL client to consume custom scalars.

- https://github.com/remorses/genql/issues/57 (hm, genql, no)
- https://github.com/helios1138/graphql-typed-client#custom-scalar-type-mapping (this might work)

Reply:
Hm, I think what we can do instead is something like this:


```ts
import * as GQLScalars from 'graphql-scalars'

expect(typeof GQLScalars.BigIntResolver.parseValue(data.bars[0].foo.someBigIntField)).toEqual('bigint')
```